lib/twine/formatters/apple.rb

```diff
@@ -25,6 +25,7 @@ def determine_language_given_path(path)
             if match[1] != "Base"
               return match[1]
             end
+            return @options[:developer_language]
```

Comment:
Thanks for the PR and information! I think I agree that this would be better for the Apple formatter. I have never run into this issue personally because we don't use the base localization for strings. We still define all languages and localization explicitly.

One suggestion: would this code be better as follows?

```
if match[1] == "Base"
  return @options[:developer_language]
else
  return match[1]
end
```

Also extra credit for a unit test, but not necessary.

Reply:
Yep, it is more clear for the reader. done !